vips/vip-304/bscmainnet.ts

```diff
@@ -0,0 +1,118 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { NETWORK_ADDRESSES } from "../../src/networkAddresses";
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const { bscmainnet } = NETWORK_ADDRESSES;
+
+export const DEV_WALLET = "0x48e9d2128321cbf75cd108321459865357c00f15";
+export const WBNB = "0xbb4CdB9CBd36B01bD1cBaEBF2De08d9173bc095c";
+export const USDC = "0x8AC76a51cc950d9822D68b83fE1Ad97B32Cd580d";
+export const USDT = "0x55d398326f99059fF775485246999027B3197955";
+export const ETH = "0x2170Ed0880ac9A755fd29B2688956BD959F933F8";
+export const BTC = "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c";
+export const vUSDC = "0xecA88125a5ADbe82614ffC12D0DB554E2e2867C8";
+export const TOKEN_REDEEMER = "0x2367bff367Bc4a83A65EEEe3b9E4D834Ff4f637e";
```

Comment:
We can use `0xC53ffda840B51068C64b2E052a5715043f634bcd`, the latest `TokenRedeemer` contract

Reply:
[bdc98c2](https://github.com/VenusProtocol/vips/pull/288/commits/bdc98c26846b4b3c99e22ab6ec9c846a118d0f8e)